lib/Doctrine/DBAL/Driver/IBMDB2/DB2Driver.php

```diff
@@ -35,6 +36,12 @@ public function connect(array $params, $username = null, $password = null, array
      */
     public function getName()
     {
+        Deprecation::triggerIfCalledFromOutside(
```

Comment:
This method seems to be called only from tests, should be `trigger()` (the same is for the rest of the implementations).